scrabble.js

```diff
@@ -1,14 +1,133 @@
-
 const Scrabble = {
+
   score(word) {
+    let wordArray = word.split('')
+    let length = wordArray.length
+    let sum = 0;
+
+    if (length === 7) {
+      sum = 50;
+    } else if (length > 7)  {
+      throw 'Word length must be less than 7';
+    } else if (length === 0) {
+      throw 'Word cannot be empty';
```

Comment:
Instead of throwing a string, you should throw an instance of `Error`. That gives you access to stack trace information. Something like this:

```javascript
if (condition) {
  throw new Error(`Invalid word ${word}`);
}
```